package.json

```
@@ -9,7 +9,10 @@
     "nohoist": [
       "**/@nomiclabs/**",
       "**/typescript",
-      "**/ts-node"
+      "**/ts-node",
+      "**/packages",
+      "**/packages/**",
+      "**"
```

Let's just ignore the deps causing the issue and keep the benefits of hoisting for all packages.

```suggestion
      "**/typechain",
      "**/@typechain/*"
```